scripts/storage-emulator-integration/tests.ts

```diff
@@ -87,41 +91,53 @@ describe("Storage emulator", () => {
 
   const emulatorSpecificDescribe = TEST_CONFIG.useProductionServers ? describe.skip : describe;
 
-  describe("Admin SDK Endpoints", function (this) {
-    // eslint-disable-next-line @typescript-eslint/no-invalid-this
-    this.timeout(TEST_SETUP_TIMEOUT);
-    let testBucket: Bucket;
-
-    before(async () => {
-      if (!TEST_CONFIG.useProductionServers) {
-        process.env.STORAGE_EMULATOR_HOST = STORAGE_EMULATOR_HOST;
-
-        test = new TriggerEndToEndTest(FIREBASE_PROJECT, __dirname, emulatorConfig);
-        await test.startEmulators(["--only", "auth,storage"]);
-      }
+  async function resetEmulatorState() {
+    if (TEST_CONFIG.useProductionServers) {
+      await testBucket.deleteFiles();
+    } else {
+      await resetStorageEmulator(STORAGE_EMULATOR_HOST);
+    }
+  }
 
-      // TODO: We should not need a real credential for emulator tests, but
-      //       today we do.
-      const credential = fs.existsSync(path.join(__dirname, SERVICE_ACCOUNT_KEY))
-        ? admin.credential.cert(readJson(SERVICE_ACCOUNT_KEY))
-        : admin.credential.applicationDefault();
+  before(async function (this) {
```

Comment:
After making all of this setup and teardown logic common, does running individual tests in this file still work? just want to confirm

Reply:
yup